examples/glue/run_pl.sh

```diff
@@ -0,0 +1,36 @@
+# Copy over transformer base
+cp -n ../transformer_base.py ./transformer_base.py
```

Comment:
Let's not copy over python files. Probably the easiest thing is to add that directory to the local path.

Reply:
Reason I chose to copy was because when you run the `isort --recursive examples templates tests src utils` command from the Makefile, it will move that line above the import for the base transformers, causing the code to fail. Any thoughts?

Reply:
@LysandreJik Do you have any idea how we might do this?

Reply:
Apparently this works:

# isort: off
gevent.monkey.patch_all()
# isort: on

https://github.com/timothycrosley/isort/issues/468

Reply:
Didn't work, unfortunately. In that issue you linked to, it says this will be implemented in version 5.0.0, but the PyPi version is 4.*

Here's what happened:

```python
import argparse
import glob
import logging
import os
import sys

import numpy as np
import torch
from seqeval.metrics import f1_score, precision_score, recall_score
from torch.nn import CrossEntropyLoss
from torch.utils.data import DataLoader, TensorDataset

from utils_ner import convert_examples_to_features, get_labels, read_examples_from_file

# isort: off
sys.path.append("../")
from transformer_base import BaseTransformer, add_generic_args, generic_train
# isort: on
```

becomes this:

```python
import argparse
import glob
import logging
import os
import sys

import numpy as np
import torch
from seqeval.metrics import f1_score, precision_score, recall_score
from torch.nn import CrossEntropyLoss
from torch.utils.data import DataLoader, TensorDataset

from transformer_base import BaseTransformer, add_generic_args, generic_train
from utils_ner import convert_examples_to_features, get_labels, read_examples_from_file


# isort: off
sys.path.append("../")

# isort: on
```

Reply:
This seems to work: https://github.com/timothycrosley/isort/issues/693#issuecomment-588987925

Reply:
I don't love this. Let's just add this to the shell script for now: 

`export PYTHONPATH="../":"${PYTHONPATH}"`

Reply:
Done :+1: